### ISSUE TITLE
Tune hyperparameters of sentiment model

### ISSUE THREAD

**Issue by sahibpreetsingh12** [sahibpreetsingh12/faaltu#8]

**Comment by sahibpreetsingh12** [comment]
Tune hyperparameters of sentiment model: Experiment with model architecture, regularization, learning rate to optimize performance.
